packages/sfui/frameworks/react/components/VsfProductCard/VsfProductCardImage.tsx

```diff
@@ -31,9 +32,9 @@ export default function VsfProductCardImage({
     // eslint-disable-next-line jsx-a11y/alt-text
     <img
       src={imageSrc}
-      width={vertical ? imageDimensionsVertical() : imageDimensionsHorizontal()}
-      height={vertical ? imageDimensionsVertical() : imageDimensionsHorizontal()}
-      className={className}
+      width={direction === VsfProductCardDirection.vertical ? imageDimensionsVertical() : imageDimensionsHorizontal()}
+      height={VsfProductCardDirection.vertical ? imageDimensionsVertical() : imageDimensionsHorizontal()}
```

Comment:
In my opinion this kinda inline conditions are hard to understand. What would you say if we change it to just.
```suggestion
      width={imageSize[direction]}
      height={imageSize[direction]}
```
Then in the component we need a function like `const imageSize = getImageSize(size);` which would be defined outside the component like that:
```
type ImageSizeByDirection = {
  [Key in `${VsfProductCardDirection}`]: number;
};

const getImageSize = (size: VsfProductCardImageProps['size']): ImageSizeByDirection => {
  switch (size) {
    case VsfProductCardSize.sm:
      return {
        horizontal: 128,
        vertical: 192,
      };

    default:
      return {
        horizontal: 192,
        vertical: 320,
      };
  }
};
```
I think this is be more readable, isn't it?

Reply:
Sure, I'll move it to some util